src/main/resources/static/cas2-domain-events-api.yml

```diff
@@ -185,10 +185,26 @@ components:
         description:
           type: string
           example: 'More information about the application has been requested from the POM (Prison Offender Manager).'
+        statusDetails:
+          type: array
+          items:
+            $ref: '#/components/schemas/StatusDetailItem'
       required:
         - name
```

Comment:
should `statusDetails` be added as a required field here? We should always return the field even if it's an empty list? (not sure myself)

Reply:
`statusDetails` has been made required in a recent commit.